prawns/governance/src/clients/subsquare/index.ts

```diff
@@ -2,8 +2,13 @@ import fetch from 'node-fetch';
 import { SubstrateNetwork } from '../../model';
 import { SubsquareDemocracyProposal } from './types';
 
+const getBaseUrl = (network: SubstrateNetwork) => {
+  const subdomain = network === SubstrateNetwork.phala ? 'khala' : network;
+  return `https://${subdomain}.subsquare.io/api/`;
+}
+
 const getDemocracyProposal = async (network: SubstrateNetwork, proposalIndex: number): Promise<SubsquareDemocracyProposal> => {
-  const response = await fetch(`https://${network}.subsquare.io/api/democracy/proposals/${proposalIndex}`);
+  const response = await fetch(`${getBaseUrl(network)}/democracy/proposals/${proposalIndex}`);
```

Comment:
Might not be an issue but this will end up with two slashes as `https://${subdomain}.subsquare.io/api//democracy...`